### DISCUSSION TITLE
How get github flavoured markdown support with mkdocs?

### DISCUSSION THREAD

**Discussion by ssbarnea** [mkdocs/mkdocs#3102] [Q&A]
As we use github a lot there are few things that I see as not working with default setup. Are there any plugins that could address these?

- render `#123` as hyperlinks to github issue tracker
- render `@username` as links to github user being mentioned
- render `mermaid` diagrams
- ability to dump github releases into a temporary changelog.md when building docs

I should mention that after reaching two dead project I found https://github.com/zopieux/pycmarkgfm which is python project that should be able to produce good rendering but that does look as being a plugin for mkdocs, so not really useful.

**Comment by pawamoy** [comment]
- See https://facelessuser.github.io/pymdown-extensions/extensions/magiclink/ for issues and users autolinking.
- See https://facelessuser.github.io/pymdown-extensions/extensions/superfences/#uml-diagram-example for Mermaid diagrams support (also note that Material for MkDocs has built-in support for Mermaid diagrams as well).
- No plugins/extensions I know of. To avoid vendor lock-in, I'd recommend maintaining a changelog.md and generating GitHub releases from it :shrug: Do you write your GitHub release notes manually?

**Reply by ssbarnea** [reply]
I am unding https://github.com/release-drafter/release-drafter to produce and update the changelogs, fully automated. Basically with few very simple API calls you can retrieve the markdown of each release note from github, still making mkdocs not choke with these is another issue. Basically the problem is related to transformation of issue mentions and user mentions into links (my first two items), everything else works fine.

**Reply by ssbarnea** [reply]
Migration from sphinx/rst to mkdocs successful... https://ansi2html.readthedocs.io is done. It required few extra steps but it did work fine https://github.com/pycontribs/ansi2html/pull/204/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2
